py3status/modules/check_tcp.py

```diff
@@ -24,24 +26,22 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 10
-    color_down = None
-    color_up = None
     format = '{host}:{port} {state}'
     host = 'localhost'
     port = 22
 
-    def check_tcp(self, i3s_output_list, i3s_config):
+    def check_tcp(self):
         response = {'cached_until': self.py3.time_in(self.cache_timeout)}
 
         sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         result = sock.connect_ex((self.host, self.port))
 
         if result == 0:
             state = 'UP'
-            response['color'] = self.color_up or self.py3.COLOR_GOOD
+            response['color'] = self.COLOR_UP or self.py3.COLOR_GOOD
```

Comment:
Confused here, isn't that supposed to be self.py3.COLOR_UP ?

Just like here : https://github.com/ultrabug/py3status/pull/411/files#diff-b153ef94eda7ed9e0be8d993f1daed17R332